about.html

```diff
@@ -57,23 +65,23 @@ <h5 class="venue">Please Contact : globalhealth@gmail.com <br>Mirpur national st
     <section class="about-section">
         <section class="about-section-a">
             <article>
-             <h2 class="about-creative-title">Creative Commons <br> Global Summit 2015 Logo</h2>
-             <img src="#" alt="">
+             <h2 class="about-creative-title">Creative Commons Global Summit 2015 Logo</h2>
+             <img src="#" alt="image">
              <p class="about-creative-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem rerum atque nulla natus.</p>
             </article>
         </section>
         <section class="about-section-b">
-            <img src="image/about-ccglobal.png" alt="">
+            <img src="image/about-ccglobal.png" alt="image">
         </section>
         <section class="about-section-c">
             <h2 class="about-img-title">See the Past CC Global Summits</h2>
             <h4 class="about-title-details">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Provident iste consectetur nesciunt.</h4>
          <div class="about-images">
             <div class="img-1">
-                <img src="image/about-img1.png" alt="">
+                <img src="image/about-img1.png" alt="image">
             </div>
             <div class="img-2">
-                <img src="image/about-img2.png" alt="">
+                <img src="image/about-img2.png" alt="image">
             </div>
          </div>
         </section>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to add a separator after the headline of the two sections in the middle so your website looks exactly as the provided design

![Screenshot_20230325_153928](https://user-images.githubusercontent.com/25227022/227727276-f18b6254-f9b6-40df-b5d3-f87c6e45e6c4.png)

|Your website|Provided design|
|-|-|
|![Screenshot_20230325_153237](https://user-images.githubusercontent.com/25227022/227727236-4e143892-8f74-42f0-9ed4-3203a46ea658.png)|![Screenshot_20230325_153914](https://user-images.githubusercontent.com/25227022/227727262-2aaa8222-6b1f-46fa-a619-818bdbe17f1b.png)|